src/language/providers/intellisense/elementItems.ts

```diff
@@ -49,17 +49,17 @@ export const elementCompletion = (definedVariables, nsPrefix) => {
       },
       {
         item: 'dfdl:assert',
-        snippetString: '<' + nsPrefix + 'assert $0',
+        snippetString: '<' + 'dfdl:' + 'assert $0',
```

Comment:
I see no advantage to gluing together these static strings.

`snippetString: '<' + 'dfdl:' + 'assert $0',`

Should be simply:

`snippetString: '<dfdl:assert $0',`

Reply:
agreed